app/forms/collection_form.rb

```diff
@@ -56,7 +45,7 @@ def cocina_model
       }
     }
 
-    reg_params[:description] = build_description if params[:collection_title].present? && params[:collection_abstract].present?
+    reg_params[:description] = build_description if params[:collection_title].present? || params[:collection_abstract].present?
```

Comment:
Can you say more about how this change, making the expected descriptive attrs more lenient, factors in?

Reply:
I believe this was a bug. However this code was never activated because we haven't had descriptive being updated in prod, so nobody noticed.

Reply:
FWIW, I don't know of a route to this form for an existing collection. Updates to collection description would go through modsulator/the datastream itself.

Reply:
It's only used for registration of new collections (via a new APO).